### ISSUE TITLE
ML Model Details

### ISSUE THREAD

**Issue by priyanshisharma** [rizwan192/Cataract-Detection-Flask-app#1]
Hello, your project caught my interest. Can you share more information about how did you come up with the ML model? as well as some performance metrics for the model? :)

**Comment by rizwan192** [comment]
Hi,
There are two models. One to extract the pupil and another for detecting the cataract.
The cataract model is developed using TensorFlow Keras. It uses a CNN and transferring learning (VGG-16). The average accuracy is almost 98%
For pupil detection, I used a python library called DLIB. Which will extract the pupil from the pictures uploaded

**Comment by priyanshisharma** [comment]
Ooh.. I saw only one model in the models folder.. which one is that? Is there somewhere that you've documented the development of your model(s)? I would like to learn.. :)
